### PR TITLE
feat(global): new backend

### PR DESCRIPTION
**Pull request by HoloPanio**
This is the new version of the backend.

### CODE REVIEW

File: .deployment.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "name": "discord-bots",
```
Comment (amitojsingh366) [inline .deployment.json]: this shouldn't be the env name! it will clash with our test env

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

**Comment by amitojsingh366** [comment]
please follow the proper pattern for commit messages *(💡TIP: you can use `yarn commit` or `yarn commit:signed` and answer the questions asked to generate a properly formatted one for you!)*

to fix your most recent commit you can run the command `git commit --amend -m "properly formatted commit message here"`

to fix your older/multiple commits please take a look at [this article](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message#amending-older-or-multiple-commit-messages)

File: .deployment.json (hunk 1 above)
Comment (HoloPanio) [reply .deployment.json]: fixed

File: .deployment.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "name": "duxcore-backend",
```
Comment (amitojsingh366) [inline .deployment.json]: i think naming it just duxcore is more suitable since we might want to host all containers under the same env

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: .deployment.json (hunk 2 above)
Comment (HoloPanio) [reply .deployment.json]: oh yeah, I forgot my bad...

**Pull request merged**